src/libs/ReportActionsUtils.js

```diff
@@ -193,6 +193,11 @@ function getSortedReportActionsForDisplay(reportActions) {
     const filteredReportActions = filterOutDeprecatedReportActions(reportActions);
     const sortedReportActions = getSortedReportActions(filteredReportActions, true);
     return _.filter(sortedReportActions, (reportAction) => {
+        // Allow all Policy Change reportActions to be displayed
+        if (reportAction.actionName && reportAction.actionName.startsWith(CONST.REPORT.ACTIONS.TYPE.POLICYCHANGELOG)) {
```

Comment:
but then we would return early because the actual `actionName` wouldn't be in the array, no?

Reply:
Discussed 1:1 a little bit, one issue is that every value in CONST.REPORT.ACTIONS.TYPE is a full string except for `POLICYCHANGELOG`. So this makes the logic a little messy since one uses `has` and another has `startsWith`. My suggestion is to add all the policy change log types to keep the logic clean and consistent.

Reply:
Updated the code!